src/sentry/static/sentry/app/views/organizationDiscover/index.jsx

```diff
@@ -92,9 +102,17 @@ const OrganizationDiscoverContainer = createReactClass({
 
   renderLoading: function() {
     return (
-      <LoadingContainer>
-        <LoadingIndicator />
-      </LoadingContainer>
+      <DiscoverContainer>
```

Comment:
Wouldn't it be better to push loading state into `Discover` since the layout is being rendered there?

Reply:
I'd rather keep this here for now so Discover doesn't have to be aware of loading state